.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+| Question      | Answer
+| ------------- | ---
+| Tickets       | <!-- URLs to GitHub or JIRA issue(s) (or N/A) -->
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | yes/no
+| Tests pass?   | yes/no
+| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)
+
+
+<!-- Replace this comment with Pull Request description -->
+
+
+#### Checklist:
+- [ ] Coding standards (`$ php-cs-fixer fix .`)
```

Comment:
... or could be `$ composer fix-cs` :)

Reply:
It's only for PHP, so for frontend we'll have to tailor our own fixer.

Reply:
If you would like to have it in PR template, just expand it anytime